Packages.props

```diff
@@ -23,7 +23,7 @@
 		<PackageReference Update="Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider" Version="1.1.1" />
 		<PackageReference Update="Microsoft.SqlServer.Management.SmoMetadataProvider" Version="170.12.0" />
 		<PackageReference Update="Microsoft.SqlServer.DacFx" Version="162.0.15-preview" />
```

Comment:
Since you are already here, do you mind updating the DacFx version with the latest here as well (the build is currently in pipeline)?